services/common/src/components/forms/RenderLargeSelect.tsx

```diff
@@ -22,7 +22,7 @@ const RenderLargeSelect: FC<LargeSelectProps> = ({
   dataSource = [],
   ...props
 }) => {
-  const selectedInData = dataSource.includes((opt) => props.selectedOption?.value === opt.value);
+  const selectedInData = dataSource.find((opt) => props.selectedOption?.value === opt.value);
```

Comment:
I kept seeing that I had like 5 copies of the `props.selectedOption` in the list, turns out that `dataSource.includes` _always returns false_- it doesn't want a predicate.
I had a hard time figuring out what exactly was the issue so random stuff like this got fixed just because it looked sus.

Reply:
Oh yeah.  That makes sense.  I've never seen `includes` used in that way.  I think it only accepts single instance of the data source passed in to work properly.